plugins/processors/ec2tagger/ec2tagger.go

```diff
@@ -418,44 +366,47 @@ func (t *Tagger) refreshLoopToUpdateTagsAndVolumes() {
 	}
 }
 
-// updateVolumes calls EC2 describe volume
-func (t *Tagger) updateVolumes() error {
-	if t.ebsVolume == nil {
-		t.ebsVolume = NewEbsVolume()
+/*
+	Retrieve metadata from IMDSv2,IMDSv1 and use these metadata to:
```

Comment:
There is no point to call out that it uses IMDSv2 and IMDSv1 because you are not explicitly calling both of them and are letting the underlying SDK determine that for you.

Reply:
Agree. Resolving it in the newest commit.